monkey/infection_monkey/exploit/sshexec.py

```diff
@@ -170,6 +191,11 @@ def _exploit_host(self) -> ExploiterResultData:
                 ssh = self.exploit_with_login_creds(port)
             except FailedExploitationError:
                 self.exploit_result.error_message = "Exploiter SSHExploiter is giving up..."
+                self.publish_exploitation_event(
```

Comment:
We already fail for specific exploitation attempts, do we need to send a failed event here?

Reply:
Break into its own PR. We can discuss there.